packages/client-core/src/admin/components/locations/AddEditLocationModal.tsx

```diff
@@ -130,80 +143,145 @@ export default function AddEditLocationModal({
     submitLoading.set(false)
   }
 
-  return (
-    <Modal
-      title={
-        location?.id ? t('admin:components.location.updateLocation') : t('admin:components.location.createLocation')
+  const unpublishLocation = async () => {
+    if (location?.id) {
+      submitLoading.set(true)
+      try {
+        await locationMutation.remove(location.id)
```

Comment:
```suggestion
        await locationMutation.remove(location.id, { query: {projectId: // need to send location id here} })
```

Getting following error when I am not not admin in MT. This is due to verifyProjectPermission hook failing (https://github.com/EtherealEngine/etherealengine/blob/e018fc9ed1c923ee1ce71c85fbd527b3d8f1e8d0/packages/server-core/src/social/location/location.hooks.ts#L250)

![image](https://github.com/user-attachments/assets/05569f21-e5f2-4144-b9b0-df25d22807ce)